### PR TITLE
fatsort: update to 1.4.2.439

### PR DESCRIPTION
**Pull request by dgilman**
###### Type(s)
- [x] enhancement

###### Tested on
macOS 10.10.5 14F2511
Xcode 7.2 7C68

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/fatsort/Portfile [hunk 1]
```diff
@@ -22,10 +22,10 @@ depends_build       port:help2man
 depends_lib         port:libiconv
 
 master_sites        sourceforge:${name}
-checksums           md5     f1232f40eba6ee9362acd9f0d5209dcf \
-                    sha1    9ae5d1ab5e4c91e6725237fab1271f881c3edb59 \
-                    rmd160  47e848fa3936d2bfc7f6e62f986055d312d809b9 \
-                    sha256  77acc374b189e80e3d75d3508f3c0ca559f8030f1c220f7cfde719a4adb03f3d
+distfiles           ${name}-${version}.tar.xz
```
Comment (mf2k) [inline sysutils/fatsort/Portfile]: Do not set the `distfiles` directly. Just use the existing command for `xz`.

`use_xz             yes`

**Comment by pmetzger** [comment]
@ionic would you mind if this went openmaintainer?

**Comment by pmetzger** [comment]
@dgilman Now LGTM.

**Comment by pmetzger** [comment]
Merged. Thanks, @dgilman!

**Pull request merged**